src/modules/pages/components/Website/Contact/Form.jsx

```diff
@@ -74,65 +73,44 @@ type Props = {|
 |};
 
 const validationSchema = yup.object().shape({
-  companyName: yup.string().required(MSG.validationTextRequired),
-  companySize: yup.string().required(MSG.validationTextRequired),
+  project: yup.string().required(MSG.validationTextRequired),
+  projectSize: yup.string().required(MSG.validationTextRequired),
   email: yup
     .string()
     .email(MSG.validationTextEmail)
     .required(MSG.validationTextRequired),
-  nameFirst: yup.string().required(MSG.validationTextRequired),
-  nameLast: yup.string().required(MSG.validationTextRequired),
+  firstName: yup.string().required(MSG.validationTextRequired),
+  lastName: yup.string().required(MSG.validationTextRequired),
   useCase: yup.string().required(MSG.validationTextRequired),
-  websiteUrl: yup.string().url(MSG.validationTextUrl),
+  website: yup.string().url(MSG.validationTextUrl),
 });
 
 const displayName = 'pages.Website.Contact.Form';
 
 const Form = ({ initialValues }: Props) => {
-  const { error, response, submitForm } = useHubspotForm({
-    formGuid: '6bcced45-41e7-40eb-b3ec-42ac313eed0a',
-    pageName: 'Contact',
-    pageUri: `https://colony.io${PAGE_CONTACT}`,
-    portalId: '4846129',
+  const { error, response, submitForm } = useDiscordWebhook({
+    webhookId: '724992469443543152',
+    webhookToken:
+      'zvCb9QZ-dhRiuQRrEbL5NmqeVOUz5nGpHbRWHTU1NNc_8WZTQFBG9asslCuAQHRHFrRB',
```

Comment:
Is it bad that we're storing this here? For anyone to read?

Reply:
I suppose this could be an env variable, at least. But similar to the hubspot forms, these will still be exposed via the network pane upon `POST`. I'll amend this commit 👍